home-manager/linux.nix

```diff
@@ -51,6 +51,18 @@
       ]);
   };
 
+  programs = {
+    bash.initExtra = ''
```

Comment:
I remember home-manager doing these tricks https://github.com/nix-community/home-manager/blob/release-24.11/modules/programs/mise.nix#L97-L109

Looks magical merging... I should learn more.